### PR TITLE
Final work on front page

### PR DESCRIPTION
**Pull request by nishahs08**
Required Design : 
image

TODO : 
1. The design needs to be checked and updated again on application. (looks different in browser and desktop application) .
2. Need to add text instead of dummy text.

Final look with LibHint (safari mac): 
image

Final look without LibHint :
image

### CODE REVIEW

File: olca-app-html/images/background.jpg
Comment (francoislerall) [inline olca-app-html/images/background.jpg:1]: Please do not include images in the commits. We try as much as possible to keep the size of the repertory small so that it is easy to clone.
The image is manually added when creating a new release.

File: olca-app-html/src/home/home.css [hunk 1]
```diff
@@ -30,7 +30,7 @@ body {
     font-family: 'TheSans', Helvetica, Arial, sans-serif;
     color: #fbfbfb;
     line-height: 1.42857143;
-    background-image: url('images/background_start_page.jpg');
+    background-image: url('images/background.jpg');
```
Comment (francoislerall) [inline olca-app-html/src/home/home.css:33]: Let's keep the same name. I guess Michael used that name to avoid confusion with any other HTML pages within openLCA.
```suggestion
    background-image: url('images/background_start_page.jpg');
```

File: olca-app-html/src/home/messages.ts [hunk 2]
```diff
@@ -6,252 +6,352 @@ const messages: Record<string, Record<string, string>> = {
...
-        "experts.title": "Make the most out of openLCA",
-        "experts.text": "GreenDelta, the creator of openLCA, offers openLCA <a id=\"services\">support services</a> via the GreenDelta helpdesk.",
-        "experts.link": "The <a id=\"lcacs\">LCA Collaboration Server</a> helps you to get the most out of working with openLCA in your team. If you do not want to run an LCA Collaboration Server yourself, GreenDelta offers you <a id=\"hosting\">hosting packages</a>."
+        "firstSteps.title": "1st steps in openLCA",
+        "firstSteps.text": "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
```
Comment (francoislerall) [inline olca-app-html/src/home/messages.ts:10]: Why did you change the text for a Lorem Ipsum?

**Review by francoislerall — CHANGES_REQUESTED** [review]
It tried to run openLCA with the HTML welcome page. It looks nice in black and white!

image

I think something could be improved though:
 - let's make the height 100% if not already the case to avoid having to scroll down.
 - the text in green is not really readable and could be placed in a smarter spot.
 - the font is a bit bigger in the example, but I guess it depends on the size of the browser so it might improve with having a 100% height.

File: olca-app-html/images/background.jpg
Comment (nishahs08) [reply olca-app-html/images/background.jpg:1]: Oh ok. noted :)

File: olca-app-html/src/home/messages.ts (hunk 2 above)
Comment (nishahs08) [reply olca-app-html/src/home/messages.ts:10]: I do not have the final draft of the text.  This was the proposed design for now. 
[olca2_welcome.docx](https://github.com/GreenDelta/olca-app/files/11200693/olca2_welcome.docx)

Comment (francoislerall) [reply olca-app-html/src/home/messages.ts:10]: OK I did not know about that. Let's keep the current texts for the time being. I think it is better to have these on `master`rather than Lorem Ipsum as we make releases that need to be clean.

Comment (nishahs08) [reply olca-app-html/src/home/messages.ts:10]: Yes even I thought so . Actually the headings are different than earlier so will be the text.  I will update the  new text before merging to the main . I will not merge with this Lorem Ipsum :) .
